### PR TITLE
Benji/ios 620 add app clip

### PR DESCRIPTION
**Pull request by Benjamindodgson**
This adds an App Clip target to the main iOS target. Everything runs, but the App Clip flow still needs work. 

Current compressed size is 1.6 MB and 4.7 MB uncompressed. The uncompressed size needs to remain below 10 MB. I think I have the entire onboarding flow added so we should be fine.

### CODE REVIEW

File: Shared/App/Lorem.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```
Comment (Benjamindodgson) [inline Shared/App/Lorem.swift:10]: I couldn't find a way around this. I needed to import UIKit in a bunch of places.

File: Shared/Coordinator/MainCoordinator.swift [hunk 2]
```diff
@@ -57,7 +57,7 @@ class MainCoordinator: Coordinator<Void> {
             }
 #elseif APPCLIP
             // Code your App Clip may access.
-            self.handleAppClip(result: launchStatus)
+          //  self.handleAppClip(result: launchStatus)
```
Comment (Benjamindodgson) [inline Shared/Coordinator/MainCoordinator.swift:60]: This I will need to revisit in future PR

File: Shared/Flows/Onboarding/OnboardingCoordinator.swift [hunk 3]
```diff
@@ -124,23 +124,9 @@ extension OnboardingCoordinator: OnboardingViewControllerDelegate {
...
-        if let cid = cid {
-            Task {
-                try await self.saveInitialConversation(with: cid)
-            }
-        }
-
         self.goToNextContentOrFinish()
     }
 
-    private func saveInitialConversation(with conversationId: String?) async throws {
```
Comment (Benjamindodgson) [inline Shared/Flows/Onboarding/OnboardingCoordinator.swift:137]: I deleted this as this seems unnecessary. there are better ways to save.

File: Shared/Models/MicroAnimation.swift [hunk 4]
```diff
@@ -12,9 +12,4 @@ enum MicroAnimation: String {
     case error = "error"
     case faceScan = "face_scan"
     case loading = "loading"
-    
-    case alertCircle = "alertCircle"
-    case checkMark = "checkmark"
```
Comment (Benjamindodgson) [inline Shared/Models/MicroAnimation.swift:17]: Not used

**Review by MartinJibber — APPROVED** [review]

**Pull request merged**